src/SFA.DAS.LevyTransferMatching.Web/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Extensions
+{
+    public static class DateTimeExtensions
+    {
+        public static string ToTaxYear(this DateTime dateTime, string format)
+        {
+            DateTime newTaxYear = new DateTime(dateTime.Year, 4, 6);
+
+            if (dateTime.Date < newTaxYear)
+            {
+                dateTime = dateTime.AddYears(-1);
```

Comment:
We probably ought to create a new variable here, to prevent modification to the parameter.

Reply:
Updated.